setup.py

```diff
@@ -13,16 +13,10 @@
 include_dirs = [
     pybind11.get_include(),
     np.get_include(),
+    "./hnswlib",
 ]
 
-# compatibility when run in python_bindings
```

Comment:
just for my understanding - why is this ok to remove?

Reply:
this was additional complexity for if you wanted to run setup.py inside the `python_bindings` directory - we don't need that.